src/scitype.jl

```diff
@@ -1,68 +1,29 @@
-# -----------------------------------------------------------------
-# scitype function (generic)
+# -----------------------------------------------------------------------------------------
+# This file introduces `scitype`, `Scitype` methods and associated fallbacks methods. 
+# It also defines some convineince methods.
+# -----------------------------------------------------------------------------------------
 
+# -----------------------------------------------------------------------------------------
+# scitype function (generic) with fallbacks.
 """
-    scitype(X)
-
-The scientific type (interpretation) of `X`, as distinct from its
-machine type, as specified by the active convention.
-
-### Examples from the MLJ convention
-
-```
-julia> using ScientificTypes # or `using MLJ`
-julia> scitype(3.14)
-Continuous
-
-julia> scitype([1, 2, 3, missing])
-AbstractArray{Union{Missing, Count},1}
-
-julia> scitype((5, "beige"))
-Tuple{Count, Textual}
-
-julia> using CategoricalArrays
-julia> X = (gender = categorical(['M', 'M', 'F', 'M', 'F']),
-            ndevices = [1, 3, 2, 3, 2])
-julia> scitype(X)
-Table{Union{AbstractArray{Count,1}, AbstractArray{Multiclass{2},1}}}
-```
-
-The specific behavior of `scitype` is governed by the active
-convention, as returned by `ScientificTypesBase.convention()`. The
-[ScientificTypes.jl
-documentation](https://alan-turing-institute.github.io/ScientificTypes.jl/dev/)
-details the convention demonstrated above.
```

Comment:
I'm assuming that this documentation is not being thrown out, but is begin moved to ScientificTypes.jl where the single argument `scitype(X)` function is first defined. 

Is that correct?

Reply:
Yes